src/utils/formUtils.js

```diff
@@ -113,3 +113,21 @@ export const getEntityTags = props => {
 
   return get(entity, ['tags', 'tagList'], []);
 };
+
+/**
+ * Formats Decorator Value according requirements
+ *
+ * @param {string} currentValue
+ * @param {string} newValue
+ * @param {boolean} isNeedToWrapInQuotes
+ * @return {string} Decorated input value
+ */
+export const formatDecoratorValue = (currentValue, newValue, isNeedToWrapInQuotes) => {
+  const decoratorValueRegExp = /"(\\.|[^"\\])*"/g;
+  const decoratorDatepickerValueRegExp = /###TODAY###|"(\\.|[^"\\])*"/g;
+  const pattern = isNeedToWrapInQuotes || currentValue.includes('###TODAY###') ? decoratorDatepickerValueRegExp : decoratorValueRegExp;
+  const containTextForReplace = currentValue.match(pattern);
+  const updatedValue = isNeedToWrapInQuotes ? `"${newValue}"` : `###${newValue}###`;
```

Comment:
I think will be better to pass final value here (`newValue = '###TODAY###'`, but not just `TODAY`), and then your `updatedValue` will look like this:
```suggestion
  const updatedValue = isNeedToWrapInQuotes ? `"${newValue}"` : newValue;
```

Reply:
done